File: 240426/007/007.py
class meet:
    def __init__(a, code, av, time):
        a.c = code
        a.a = av
        a.t = time




c, a, t = input().split()

abc = meet(c,a,t)

print(f"secret code : {abc.c}")
print(f"meeting point : {abc.a}")
print(f"time : {abc.t}")

File: 241107/007/007.java
class a{
    String code;
    String point;
    Integer time;

    public a(String code, String point, Integer time){
        this.code = code;
        this.point = point;
        this.time = time;
    }

}

public class Main {
    public static void main(String[] args) {

        a codetree = new a("codetree", "L", 13);
        System.out.println("secret code : " + codetree.code);
        System.out.println("meeting point : " + codetree.point);
        System.out.println("time : " + codetree.time);
        // 여기에 코드를 작성해주세요.
    }
}